src/ZoneServer/World/WorldManager.cs

```diff
@@ -24,6 +24,12 @@ public class WorldManager
 		// Unique handles for buffs
 		private int _buffhandles = 0;
 
+		// Unique handles for skills
```

Comment:
I assume you tested this and it worked, but just to be sure, do these two use their own id ranges or should they perhaps be part of the entities?

Reply:
I don't know the answer to this one.

Reply:
The correct answer is most likely part of entities. What I thought were skill handles turned out to be "pad" handles, but since our pads are NPCs with triggers, you can probably avoid using them.

Reply:
Do we know if pads are their own thing? They are mentioned time and time again in the client data, but I don't think I've seen anything yet that would clarify what they are exactly. Are they monsters as well? A different type of actor?

If these skill handles, that I've also been wondering about, are pad handles, I would agree that they're probably some form of entity and should go into that pool. Or rather, the handle will just be the handle of the Monster, right? And the effects probably too? I guess we're actually already handling those that way, like in Heal.